spec/syntax.md

```diff
@@ -300,7 +305,7 @@ U+007B LEFT CURLY BRACKET `{`, and U+007D RIGHT CURLY BRACKET `}`
 MUST be escaped as `\\`, `\{`, and `\}` respectively.
 
 In the ABNF, _text_ is represented by non-empty sequences of
-`simple-start-char`, `text-char`, and `escaped-char`.
+`simple-start-char`, `text-char`, `escaped-char`, and `s`.
```

Comment:
You're adding this because simple messages can start with non-`simple-start-char` whitespace? Would we be better moving the optional whitespace into that construct?

Reply:
I opted to instead remove `s` from `simple-start-char`; the overall effect should be the same, but it keeps the `-char` rule as a single character.

Reply:
Oh, it's better than that: `simple-start-char` was not necessarily a single char previously because `s` is `1*`. So I'd say it makes the production single-char as originally intended and makes the start char clear that `.` isn't included.